src/Groups/homomorphisms.jl

```diff
@@ -741,12 +771,14 @@ end
     GrpAbFinGen(G::T) where T <: GAPGroup
     PcGroup(G::T) where T <: Union{GAPGroup, GrpAbFinGen}
     pc_group(G::T) where T <: Union{GAPGroup, GrpAbFinGen}
+    SubPcGroup(G::T) where T <: Union{GAPGroup, GrpAbFinGen}
+    sub_pc_group(G::T) where T <: Union{GAPGroup, GrpAbFinGen}
```

Comment:
Do we really want/need this? Maybe `SubPcGroup` on a `PcGroup` could be useful, in case one needs the "full group as a subgroup", but other than that?